src/app/forgotPassword/FormComponent.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import React, { useState } from "react";
+import { Formik, Form, Field } from "formik";
+import { Input, ButtonPrimary, FormItem } from "@/components/atoms";
+import * as Yup from "yup";
+import { IForgotPassword } from "@/types/auth";
+import Auth from "@/service/auth.service";
+
+const FormComponent = () => {
+  const [loading, setLoading] = useState(false)
+  const SignUpValidationSchema = Yup.object().shape({
+    email: Yup.string().email("email inválido").required("requerido"),
+  });
+
+  const initialValues: IForgotPassword = {
+    email: "",
+  };
+
+  const handleFormSubmit = async (values: IForgotPassword) => {
+    setLoading(true);
+    try {
+        await Auth.forgotPassword({
```

Comment:
Show comments on [this PR](https://github.com/Agility-Creative-Solutions/ez2live-app/pull/14)